src/modules/itemCounter.js

```diff
@@ -0,0 +1,9 @@
+const cardItemCounter = async (cardElements) => {
+  if (!cardElements) {
+    return 0;
+  }
+  const itemAmount = cardElements.length;
+  return itemAmount;
+};
```

Comment:
- Great job for implementing the counter function, I think you have to count the actual dom elements instead of getting the API length as mentioned in the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md) 👍 
![Screenshot 2023-06-30 071350](https://github.com/Diegogagan2587/JavaScript-Capstone-Project/assets/52010789/41cfef40-a6ba-4dea-8de8-d007d108a50a)

Reply:
@binhussen thanks so much for reviewing, I would like to provide some clarification, here the Variable **cardElements** is actually a node list we are getting from the **DOM**, We are using async because first, we need to wait for the API to get the data, then we display everything into the HTML and then, we have created the function cardItemCouter which receives a node list from the DOM, so... then we document.querySelectorAll("element") and then we pass the result to cardItemCounter();